toolkit/scss/forms/_combinations.scss

```diff
@@ -13,10 +13,6 @@ $border-width: $gutter / 6;
   padding-top: $gutter-two-thirds;
   margin: -$gutter 0 0 0;
 
-  &:not(:last-of-type) {
-    padding-bottom: $gutter;
-  }
-
```

Comment:
I'm not sure about cutting out these rules.  There are two things it seems to make worse right now.

| Before | After |
|--------|-------|
| ![screen shot 2017-04-21 at 16 38 59](https://cloud.githubusercontent.com/assets/2454380/25285208/5950473a-26b1-11e7-89a1-ec4cfe28f7e5.png) | ![screen shot 2017-04-21 at 16 40 41](https://cloud.githubusercontent.com/assets/2454380/25285219/5f1622ac-26b1-11e7-88ff-27f35488a931.png)  |
| ![image](https://cloud.githubusercontent.com/assets/2454380/25285330/bd771946-26b1-11e7-9531-eb0732eb57bc.png) | ![screen shot 2017-04-21 at 16 42 08](https://cloud.githubusercontent.com/assets/2454380/25285346/c9c7837a-26b1-11e7-8db6-fb20e6678dfb.png) |

In the first case, we lose spacing between the first and second questions and we break the continuous grey border.  
In the second case, the follow-up questions (once they're both opened) look a bit too squashed together.